src/depext.ts

```diff
@@ -0,0 +1,18 @@
+import * as core from "@actions/core";
+import { exec } from "@actions/exec";
+import * as path from "path";
+
+export async function installDepextWindows(
+  ocamlVersion: string
+): Promise<void> {
+  core.startGroup("Install depext");
+  await exec("opam", ["install", "opam-depext", "depext-cygwinports"]);
```

Comment:
Should we remove `opam-depext` when 2.1 for Windows is released?

Reply:
No - opam 2.1 doesn't have Windows support for depext _at all_ - it does work with a minor reconfiguration as a drop-in for opam-repository-mingw's opam. I don't know if there's any plan to release a Windows opam 2.1 - there certainly is no plan for an official release (we're aiming for official Windows support in opam 2.2)

Reply:
Oh, I see.